lock/src/main/java/org/infinispan/lock/impl/lock/ClusteredLockImpl.java

```diff
@@ -348,36 +348,33 @@ public void viewChange(ViewChangedEvent event) {
        * @param possibleOwners
        */
       private void forceUnlockForLeavingMembers(Set<Object> possibleOwners) {
-         clusteredLockManager.execute(() -> {
+         if (trace) {
```

Comment:
This diff looks all mangled. But basically I just moved everything over a bit to the left. No actual code change besides the removal of the use of the execute invocation.